src/java.logging/share/conf/logging.properties

```diff
@@ -2,15 +2,15 @@
 #  	Default Logging Configuration File
 #
 # You can use a different file by specifying a filename
-# with the java.util.logging.config.file system property.  
-# For example java -Djava.util.logging.config.file=myfile
+# in the java.util.logging.config.file system property.
```

Comment:
Unless the `java.util.logging.config` property contains more than one item of information, the original "with" is more appropriate here.

Reply:
Is it because in this case "with" is similar to "by" and "using" in the sense that "with" conveys that the system property acts as a "tool" with which the effect, specifying a filename, is achieved?